arxiv/metadata/metacheck.py

```diff
@@ -169,14 +216,14 @@ def language_is_not_english(s: str) -> bool:
         return False

```

Comment:
This comment is concerning `all_brackets_balanced` - do we reect titles with smilies, like
`The social reaction to ;-)`
?

Reply:
Yes, this code does reject smilies.